decepticonlp/metrics/temporal_metrics.py

```diff
@@ -0,0 +1,55 @@
+import torch
+import torch.nn as nn
+
+
+class RankCharacters:
+    """
+        Accepts a feature vector torch tensor and outputs a temporal ranking of characters
+        inputs of shape (number of words,feature_vector_size)
+    """
+
+    def temporal_score(self, sentence):
+        """ 
+            Considering a input sequence x1,x2,...,xn
+            we calculate T(xi) = F(x1,x2,...,xi) - F(x1,x2,...,xi-1)
```

Comment:
Docstrings in our preferred format needed here too? Including information about params and return values?